test/v2/unit/recovery.js

```diff
@@ -47,7 +47,7 @@ describe('Recovery:', function() {
         walletPassphrase: TestV2BitGo.V2.TEST_WALLET1_PASSCODE,
         recoveryDestination: '2NB5Ynem6iNvA6GBLZwRxwid3Kui33729Nw',
         scan: 5,
-        ignoreAddressTypes: ['p2wsh', 'pwshP2wsh']
+        ignoreAddressTypes: ['p2wsh', 'p2shP2wsh']
```

Comment:
Why do we ignore segwit addresses in our recovery tests? Do we have tests for this elsewhere?

Reply:
I believe it is because we only mock out p2sh txes for recovery tests. Again, I think fixing this is out of scope for this PR and for native segwit generally.